scripts/populateDatabase.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+$propertyUrl = "https://dev.maydenacademy.co.uk/resources/property-feed/properties.json";
+$statusUrl = "https://dev.maydenacademy.co.uk/resources/property-feed/statuses.json";
+$typeUrl = "https://dev.maydenacademy.co.uk/resources/property-feed/types.json";
+
+$database = getArmadilloEstates();
+
+$propertyArray = getApiData($propertyUrl);
+$statusArray = getApiData($statusUrl);
+$typeArray = getApiData($typeUrl);
+
+$database->query("TRUNCATE TABLE `properties`");
```

Comment:
these 3 truncates and foreaches could probably be done in a foreach? make it more automated